srcjs/input_binding_select.js

```diff
@@ -84,9 +85,17 @@ $.extend(selectInputBinding, {
             callback();
           },
           success: function(res) {
```

Comment:
Can you add a comment that shows an example the structure of `res`?

Reply:
Can you explain that `success` is called after the options are added, but before the groups, and that groups need to be added manually?